frame/nomination-pools/src/lib.rs

```diff
@@ -721,9 +727,13 @@ impl<T: Config> BondedPool<T> {
 	}
 
 	fn is_destroying_and_only_depositor(&self, alleged_depositor_points: BalanceOf<T>) -> bool {
-		// NOTE: if we add `&& self.member_counter == 1`, then this becomes even more strict and
-		// ensures that there are no unbonding members hanging around either.
-		self.is_destroying() && self.points == alleged_depositor_points
+		// we need to ensure that `self.member_counter == 1` as well, because the depositor's
+		// initial `MinCreateBond` (or more) is what guarantees that the ledger of the pool does not
+		// get killed in the staking system, and that it does not fall below `MinimumNominatorBond`,
+		// which could prevent other non-depositor members from fully leaving. Thus, all members
+		// must withdraw, then depositor can unbond, and finally withdraw after waiting another
+		// cycle.
+		self.is_destroying() && self.points == alleged_depositor_points && self.member_counter == 1
```

Comment:
So without this the depositor could initiate the unbonding too early?

Reply:
The depositor must stay in the pool until the last member has called `withdraw`. This ensures that the pool, as a nominator, has enough stake that everyone can call `withdraw`. Once the depositor is ready to leave, they must also explicitly `chill` the pool.

More on this in #11417